index.js

```diff
@@ -150,13 +157,13 @@ const buildVues = (callback) => {
 };
 
 VueBuilderPlugin.prototype.apply = (compiler) => {
-  compiler.plugin('run', (compilation, callback) => buildVues(callback));
-  compiler.plugin('watch-run', (compilation, callback) => buildVues(callback));
+  compiler.plugin('run', (compilation, callback) => buildVues(callback, compiler));
+  compiler.plugin('watch-run', (compilation, callback) => buildVues(callback, compiler));
 
   compiler.plugin('after-compile', (compilation, callback) => {
-    // eslint-disable-next-line no-param-reassign
     compilation.fileDependencies = Array.from(compilation.fileDependencies).filter((file) => {
-      if (file.slice(-4) === '.vue') {
+
+      if (createdFiles.some(x => x === file)) {
```

Comment:
Wouldn't an `includes` or `indexOf` be faster?

Reply:
@CKGrafico What about this comment?

Reply:
yes https://jsperf.com/includes-vs-some one minute

Reply:
check now :)